### ISSUE TITLE
Duplication between EasyConfigs that have the same name and different capitalization

### ISSUE THREAD

**Issue by mboisson** [easybuilders/easybuild-easyconfigs#11824]
I just found
https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/g/grace
and
https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/g/Grace

which are the same, with different capitalization

@boegel

**Comment by mboisson** [comment]
For completeness, I also extracted the list of EasyConfigs that have the same name, just with different capitalization : 
```
      2 a/arb
      2 a/arrow
      2 a/assimp
      2 b/bamm
      2 b/blasr
      2 c/casper
      2 c/charmm
      2 c/check
      2 g/grace
      2 g/graphviz
      2 h/hyperopt
      2 i/ioapi
      2 l/libpsml
      2 m/magma
      2 n/nanofilt
      2 n/ncl
      2 o/orca
      2 x/xmlf90
```

**Comment by SebastianAchilles** [comment]
`Grace` was fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/14493

**Comment by Micket** [comment]
Discussed brielfy on todays meeting; i wanted to give an update. Current offenders are:
- a/arb - Arb vs ARB, but ARB is just 2 patches and should just be archived. Trivial.
- b/bamm - BAMM vs BamM, neither have been updated in some time and will be archived.
- b/blasr - BLASR vs blasr; but blasrc just has a leftover patch should be moved or archived
- c/charmm - CHARMM vs charmm same software
- c/check - check vs Check same software
- g/graphviz - graphviz vs Graphviz - graphviz is the python package, it should just get a suffix. It actually depends on Graphviz!
- h/hyperopt - hyperopt vs Hyperopt same software
- l/libpsml - libpsml vs libPSML same software
- m/magma -  magma vs MAGMA MAGMA should have a suffix to indicate it's the gene analysis tool
- n/nanofilt - NanoFilt vs nanofilt same software
- n/ncl - ncl vs NCL: ncl should probably be renamed "NEXUS" or something

**Comment by lexming** [comment]
`Hyperopt` was fixed in #16697

**Comment by Micket** [comment]
What would everyone prefer for MAGMA? I suggest
1. MAGMA-gene-analysis
2. MAGMA-CTGlab
3. MAGMA-CNCR

**Comment by lexming** [comment]
All name duplicates have been fixed and we added a unit test in #17303 to catch any new ones.

**Comment by branfosj** [comment]
All fixed (see the merged PRs listed above) and a test added in #17303 to avoid adding more.

**Comment by lexming** [comment]
@branfosj Ha! I won! :smile:
